apps/src/javalab/ControlButtons.jsx

```diff
@@ -31,14 +29,6 @@ export default function ControlButtons({
           isDisabled={disableRunButtons}
           id="javalabRun"
         />
-        <JavalabButton
-          text={isTesting ? i18n.stopTests() : i18n.test()}
-          icon={<FontAwesome icon="flask" className="fa" />}
-          onClick={toggleTest}
-          isHorizontal
-          style={{...styles.button.all, ...styles.button.white}}
```

Comment:
If we are planning on adding back this button (which I believe we are eventually?) it might be easier to add a `display: none` to its style so that the props/logic stay in place? Or could put it behind another prop that is set to false for now (ex. `{showTestButton && <JavalabButton ... }`)

Reply:
Sure! I'll add a prop